code/modules/events/blob.dm

```diff
@@ -14,11 +14,9 @@
 	var/mob/C
 	if(candidates.len)
 		C = pick(candidates)
-		Blob = new /obj/effect/blob/core(T, 200)
-		Blob.create_overmind(C.client, 1)
-		spawn(30)
-			for(var/i = 1; i < rand(3, 6), i++)
-				Blob.process()
+		Blob = new /obj/effect/blob/core(T, 200, C.client, 2, 0)
+		for(var/i = 1; i < rand(3, 6), i++)

```

Comment:
While you're here, you can fix this awful line, which re-evaluates the `rand` with each iteration. `for(var/i in 1 to rand(2,5))` would do what this is *intended* to do... though you might be better off just making it `1 to 5`, since randomizing it at all seems pointless.